manuscript/manuscript.Rmd

```diff
@@ -54,24 +54,23 @@ editor_options:
+Applications of ML in ecology have begun to illustrate the promise of two methods of ML: *supervised learning* [@joseph2020] and *unsupervised learning* [@unsupervised], but have so far largely overlooked the third and possibly most promising approach in the ML triad: *reinforcement learning* (RL).
 Three features distinguish RL from other ML methods in ways that are particularly well suited to addressing global ecological change issues:
 
 1) RL is explicitly focused on the task of selecting actions in an uncertain and changing environment to maximize some objective,
 2) RL does not require massive amounts of representative sampled historical data,
 3) RL approaches easily integrate with existing ecological models and simulations, which may be our best guide to understanding and predicting future possibilities.
 
 Despite relevance to uncertainty and decision making that could make RL uniquely well suited for ecological and conservation problems, it has so far seen little application in this area.
-To date, the uncertain and changing environments considered by RL research have largely been drawn from examples in robotic movement and video games.
-RL agents have recently achieved superhuman performance on classic Atari games [@DQN], while Deepmind’s RL-trained AlphaZero has defeated the world's grandmasters in Go, Chess, and Shogi [@alphazero].
-Complex environmental problems share many similarities to these games: the need to plan many moves ahead, accept short-term costs to achieve long-term benefit, to account for uncertainty and respond with contingency to the unexpected.
-Of course real ecosystems are also far more complex than a game of chess or MoonLander -- but can the same be said of our efforts at finding the best strategy?
+To date, the problems considered by RL research have largely been drawn from examples in robotic movement and games like Go and Starcraft [@lillicrap; @alphazero; @alphastar]
+Complex environmental problems share many similarities to these tasks and games: the need to plan many moves ahead given a large number of possible outcomes, to account for uncertainty and to respond with contingency to the unexpected.
+RL agents typically develop their strategies by interacting with simulators, a practice that should not be unsettling to ecologists since learning from simulators is common across ecology.
 Rich, processes-based simulations such as the SORTIE model in forest management [@sortie], Ecopath with Ecosim in fisheries management [@ecopath], or global circulation models (GCMs) in climate [@gcm] are already used to explore scenarios and inform ecosystem management. 
-Even simpler models are frequently employed throughout ecology to guide understanding and response.
+Even simpler simulators are frequently employed throughout ecology to guide understanding and response.
```

Comment:
I really think we want to pivot back to the use of term "model" here.  I think the point of the examples like sortie and ecopath was really just to warm up ecological readers to the idea that we already use 'simulators' in some places, but we want to pivot back to more familiar territory in that most readers are much more familiar with the use of 'ecological models'.  We might need some example or more context to show that we're trying to invoke the 'everyday models' of ecology (logistic growth, SIR, Lotka Volterra, etc etc) and not just things that are individual-based simulations ("IBMs or "agent-based models").  Hmm...